apps/firehose_to_es_processor/lib/gzip_stream_reader.py

```diff
@@ -0,0 +1,65 @@
+import io
+import zlib
+from enum import Enum
+from itertools import chain
+
+
+class GzipStreamReader(io.RawIOBase):
+
```

Comment:
Why not https://docs.python.org/3/library/gzip.html#gzip.GzipFile? This also presents a stream.

Reply:
Ah, I missed this. Will simplify.